docs/Documentation/openvino_ecosystem.md

```diff
@@ -0,0 +1,75 @@
+# OpenVINO Ecosystem Overview {#openvino_ecosystem}
+
+
+OpenVINO™ is not just one tool. It is part of an expansive ecosystem of utilities, frameworks, addons, etc. They all provide a comprehensive workflow for developing deep learning solutions. Learn more about each of them to reach the full potential of OpenVINO™.
+
+### OpenVINO™ Model Server (OVMS)	   
+A scalable, high-performance solution for serving deep learning models optimized for Intel architectures. The server uses Inference Engine libraries as a backend and exposes gRPC and HTTP/REST interfaces for inference that are fully compatible with TensorFlow Serving.
+
+More resources:
+* [documentation](https://docs.openvino.ai/latest/openvino_docs_ovms.html)
```

Comment:
```suggestion
* [OpenVINO documentation](https://docs.openvino.ai/latest/openvino_docs_ovms.html)
```

Reply:
The entire document assumes the label of "documentation" under each item. I don't think additional label for the same domain is necessary.

Reply:
Sure, we'll revise this one multiple times and the final version will appear at some point ;)